app/src/main/java/com/dessalines/thumbkey/ui/components/settings/behavior/BehaviorActivity.kt

```diff
@@ -65,25 +65,20 @@ fun BehaviorActivity(
 
     val settings by appSettingsViewModel.appSettings.observeAsState()
 
-    var minSwipeLengthState by remember { mutableStateOf((settings?.minSwipeLength ?: DEFAULT_MIN_SWIPE_LENGTH).toFloat()) }
```

Comment:
These remembers were pointless, but didn't cause any problems since the app DB was always initialized before going to these routes.